src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -327,6 +328,32 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
             return Text(runner.GetFormattingInfo(), irContext, args, runner.CancellationToken);
         }
 
+        public static FormulaValue TextToBlob(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            runner.CancellationToken.ThrowIfCancellationRequested();
+
+            StringValue sv = (StringValue)args[0];
+            IResourceManager resourceManager = runner.FunctionServices.GetService<IResourceManager>();
+
+            if (resourceManager == null)
+            {
+                return new ErrorValue(irContext, new ExpressionError()
+                {
+                    Message = $"Cannot convert {args[0].Type._type} to Blob, missing resource manager.",
+                    Span = irContext.SourceContext,
+                    Kind = ErrorKind.InvalidArgument
+                });
+            }
+
+            return new BlobValue(resourceManager, new StringResourceElement(resourceManager, sv.Value).Handle);

```

Comment:
>StringResourceElement [](http://example.com/codeflow?start=54&length=21)

We should have strings here

Reply:
Why?
A blob could have multiple forms: string, base64string, byte[], ...

Reply:
A blob is semantically a list of bytes. If we pass a string, how do you get its equivalent byte[]? We can assume some encoding such as UTF-8, but that is not going guaranteed to be correct (some system may be expecting some other encoding, UTF-8 does not offer the best performance for many Eastern languages, ...). We could add an encoding argument, but that would make the interface more complicated. Ideally we should let the encoding of data to the resource manager itself.